### PR TITLE
feat: bound the measure of a set by the integral of a function, from above and from below

### PR DESCRIPTION
**Pull request by sgouezel**
Also weaken some T2 space assumptions in some integration lemmas.

### CODE REVIEW

File: Mathlib/MeasureTheory/Integral/Lebesgue.lean [hunk 1]
```diff
@@ -758,19 +758,31 @@ theorem lintegral_rw₂ {f₁ f₁' : α → β} {f₂ f₂' : α → γ} (h₁
...
+  refine le_iSup_of_le g (le_iSup_of_le this (le_of_eq ?_))
+  rw [lintegral_restrict, SimpleFunc.lintegral]
+  congr with t
+  by_cases H : t = 0
+  · simp [H]
+  congr with x
+  simp only [mem_preimage, mem_singleton_iff, mem_inter_iff, iff_self_and]
+  intro hx
+  contrapose! H
+  rw [← hx]
```
Comment (hrmacbeth) [inline Mathlib/MeasureTheory/Integral/Lebesgue.lean:775]:
```suggestion
  rintro rfl
  contrapose! H
```

File: Mathlib/MeasureTheory/Integral/SetIntegral.lean [hunk 2]
```diff
@@ -810,6 +810,37 @@ theorem set_integral_nonpos_le {s : Set α} (hs : MeasurableSet s) (hf : Strongl
...
+    μ s ≤ ENNReal.ofReal (∫ x, f x ∂μ) := by
+  rw [ofReal_integral_eq_lintegral_ofReal f_int f_nonneg]
+  apply meas_le_lintegral₀
+  · exact ENNReal.continuous_ofReal.measurable.comp_aemeasurable f_int.1.aemeasurable
+  · intro x hx
+    simpa using ENNReal.ofReal_le_ofReal (hs x hx)
+
+lemma integral_le_measure {f : α → ℝ} {s : Set α}
+    (hs : ∀ x ∈ s, f x ≤ 1) (h's : ∀ x ∈ sᶜ, f x ≤ 0) :
+    ENNReal.ofReal (∫ x, f x ∂μ) ≤ μ s := by
```
Comment (hrmacbeth) [inline Mathlib/MeasureTheory/Integral/SetIntegral.lean:824]: You have called this one `integral_le_measure` but the one in lintegral `lintegral_le_meas`, is the inconsistency `meas`/`measure` due to pre-existing different patterns in the two files?

**Review by hrmacbeth — COMMENTED** [review]
bors d+

File: Mathlib/MeasureTheory/Integral/SetIntegral.lean (hunk 2 above)
Comment (sgouezel) [reply Mathlib/MeasureTheory/Integral/SetIntegral.lean:824]: I think `meas` is a shortcut that has been used at some point, but I definitely prefer the barely longer and much clearer `measure`. I should probably make another PR to uniformize this.

**Pull request merged**